src/main/java/Duke.java

```diff
@@ -1,10 +1,312 @@
+            "    ____________________________________________________________";
+    private final String PADDING = "      ";
+
+    private enum Commands {
+        EXIT("BYE"), DELETE("DELETE "), LIST("LIST"),
+        DONE("DONE "), TODO("TODO "), EVENT("EVENT "),
+        DEADLINE("DEADLINE "), DELETEALL("DELETE ALL");
+        private final String str;
+        Commands(String str){
+            this.str = str;
+        }
+        public String getString() {
+            return this.str;
+        }
+    }
+
+    private final String MESSAGE_TEMPLATE = HORIZONTAL_LINE + NEW_LINE + PADDING + "%s"
+            + NEW_LINE + HORIZONTAL_LINE + NEW_LINE + NEW_LINE;
+    private final String MESSAGE_TEMPLATE_VERBAL = HORIZONTAL_LINE + NEW_LINE + PADDING + "Deuk: %s"
+            + NEW_LINE + HORIZONTAL_LINE + NEW_LINE + NEW_LINE;
+    private final String MESSAGE_TEMPLATE_ERROR = HORIZONTAL_LINE + NEW_LINE + PADDING
+            + "☹ OOPS!!! %s"
+            + NEW_LINE + HORIZONTAL_LINE + NEW_LINE + NEW_LINE;
+    private final List<Task> storageList = new ArrayList<>();
+
+    public void init() {
+        sayHello();
+        try {
+            loadTasksFromDisk();
+        }
+        catch (DukeDataFolderException ex) {
+            printError(ex.getMessage());
+            File dir = new File(DUKE_DATA_DIR_PATH.toUri());
+            Boolean isCreated = dir.mkdir();
+            if (isCreated) {
+                echoBack("Successfully created Deuk Data Folder");
+                try {
+                    FileWriter fw = new FileWriter(DUKE_DATA_FILE_PATH.toString());
+                    fw.close();
+                }
+                catch (IOException err) {
+                    printError(err.getMessage());
+                }
+            }
+            else {
+                printError("Failed to create Deuk Data Folder");
+            }
+        }
+        catch (DukeException ex) {
+            printError(ex.getMessage());
+        }
+        catch (FileNotFoundException ex) {
+            printError("Missing Deuk Data File!" + NEW_LINE + PADDING +
+                "Creating new Deuk Data File..."
+            );
+            try {
+                FileWriter fw = new FileWriter(DUKE_DATA_FILE_PATH.toString());
+                fw.close();
+            }
+            catch (IOException err) {
+                printError(err.getMessage());
+            }
+
+        }
+
+
+        while(true) {
+            if (SC.hasNext()) {
+                String input = SC.nextLine().trim();
+                if (input.toUpperCase().equals(Commands.EXIT.getString())) { // to make command case-insensitive
+                    sayGoodbye();
+                    break;
+                }
+                if (input.toUpperCase().equals(Commands.LIST.getString())) {
+                    displayStorageList();
+                }
+                else if (input.length() >= 5 && input.substring(0,5).toUpperCase().equals(Commands.DONE.getString())) {
```

Comment:
Perhaps you can try to avoid putting all the processing of the input in the predicate expression for the `if ... else`, you can try to have another function deal with the parsing and isolating user input and just return what you want here instead :D